provider/datagen/src/transform/cldr/cldr_serde/ca.rs

```diff
@@ -105,7 +105,7 @@ symbols!(
     ["midnight", midnight, Option<Cow<'static, str>>],
 );
 
-#[derive(PartialEq, Debug, Deserialize)]
+#[derive(PartialEq, Debug, Deserialize, Clone)]
```

Comment:
Question: For the structs where you had to add `Clone` impls, did you check whether it was easy to convert the transformers to working on references?